app/controllers/manifestation_controller.rb

```diff
@@ -252,13 +252,11 @@ def dict_entry
   end
 
   def read
-    @m = Manifestation.joins(:expression).includes(:expression).find(params[:id].to_i)
-    if @m.nil?
```

Comment:
removed ineffective check (@m cannot be nil as in this case RecordNotFound is thrown)